packages/amplify-provider-awscloudformation/lib/initializer.js

```diff
@@ -66,6 +67,13 @@ async function run(context) {
         spinner.fail('Root stack creation failed');
         throw e;
       });
+  } else if (
```

Comment:
Can you add a comment out here as to when this code would be executed? (For readability purproses)

Reply:
comment added